.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ orbs:
               name: Prepare build environment
               command: |
                 echo "export BUILD_NUMBER=`/home/circleci/repo/bin/generate-build-number`" >> $BASH_ENV
-                echo "export PATH=/home/circleci/.local/bin:$PATH" >> $BASH_ENV
```

Comment:
Question: Changing the quotes here will cause `$PATH` to not be expanded, is that intended?